src/TodoListItem.js

```diff
@@ -1,20 +1,22 @@
 import React from 'react';
+import style from "./TodoListItem.module.css";
 
 const TodoListItem = ({ todo, onRemoveTodo, todoCompletion }) => {
     if (!todo) {
-        return <li>Error: Todo item is missing or incomplete.</li>;
+        return <li className={style.ListItemError}>Error: Todo item is missing or incomplete.</li>;
     }
     const {title, id, completed} = todo;
 
     return (
-        <li style={{textDecoration: completed ? "line-through" : "none"}}>
+        <li className={{textDecoration: completed ? style.ListItemCompleted : style.ListItemNotCompleted}}>
```

Comment:
So this isn't working the way you'd expect because you're using in-line styling to specifically target the `textDecoration` attribute, but your ternary returns your css selectors that both contain two attributes. Making the following change will apply your selectors properly 
```suggestion
        <li className={completed ? style.ListItemCompleted : style.ListItemNotCompleted}>
```